go.sum

```diff
@@ -40,6 +40,10 @@ github.com/dgrijalva/jwt-go v3.2.0+incompatible/go.mod h1:E3ru+11k8xSBh+hMPgOLZm
 github.com/dgryski/go-sip13 v0.0.0-20181026042036-e10d5fee7954/go.mod h1:vAd38F8PWV+bWy6jNmig1y/TA+kYO4g3RSRF0IAv0no=
 github.com/digitalocean/godo v1.32.0 h1:ljfhYi/IqDYiZcBYV7nUPzw1Q7NlmPSFDtI69UeRThk=
 github.com/digitalocean/godo v1.32.0/go.mod h1:iJnN9rVu6K5LioLxLimlq0uRI+y/eAQjROUmeU/r0hY=
+github.com/digitalocean/godo v1.32.1-0.20200320152931-d8bc2d10f11e h1:xUc6id4Kfm7j4A7FRlijPLWP4Zj8gTyQ/UVJwa0LDrI=
```

Comment:
Is this 1.32.1 stuff leftover from when you were developing against an unreleased `godo` version, or is this right?

Reply:
yup i must have forgotten to `go mod tidy`, i ran `go mod vendor` directly, just noticed that there's a `make vendor` for this. will fix